src/libdrakvuf/libdrakvuf.h

```diff
@@ -223,6 +223,34 @@ struct drakvuf_trap {
     void *data;
 };
 
+
+////////////////////////////////////////////////////////////////////////////
+
+// IMHO these definitions must be placed within another file, named
+// libdrakvuf-windows.h or something similar
+
```

Comment:
I'm not 100% on your side ;)
The idea of placing these definitions on libdrakvuf.h was making available for all of the plugins.

So, for instance, the plugin can use it this way:
`drakvuf_obj_ref_by_handle( drakvuf, info, eproc, handle,                       ObjManagerProcessObject, &obj );`

Makign the plugin a little bit more readable